src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -13,6 +13,8 @@ internal class LocalizableStrings
 
         public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to add.";
 
+        public const string CmdPackageDescription = "Name of the package to be added.";
```

Comment:
@nguerrera and @piotrpMSFT : As you can see above, there was no existing name for the argument as it was sitting in Additional arguments. 

I can reuse the existing string for the description, but it will be difficult to add the argument without a name for the argument - 

` Argument(string name, string description, bool multipleValues);`